script/script.js

```diff
@@ -82,28 +86,80 @@ function removeThis(e){
 }
 
 
-function storeIdea (id, card) {
-	localStorage.setItem(id, JSON.stringify(card));
-}
 
 $('.main-title , .idea-input').on('keyup', checkValue); 
 
-function checkValue(e) {
-	if (e.keyCode === 13 && ($('.main-title').val() !== '' && $('.idea-input').val() !== '')){
+function checkValue(e) {;
+	if (e.keyCode === 13 && $('.main-title').val() !== '' && $('.idea-input').val() !== ''){
+		e.preventDefault();
 		addIdea();
```

Comment:
If all this function does is prevent blank cards than we shouldn't need addIdea() right?